--- origsrc/bless-0.5.2/builder/Makefile.am	2007-08-25 02:54:21.000000000 -0500
+++ src/bless-0.5.2/builder/Makefile.am	2007-09-02 14:30:52.625000000 -0500
@@ -1,4 +1,4 @@
-SRC= $(wildcard *.cs)
+SRC= $(wildcard $(srcdir)/*.cs)
 
 EXTRA_DIST= $(SRC)
 EXE= bless-builder.exe
--- origsrc/bless-0.5.2/configure.ac	2007-08-25 03:07:31.000000000 -0500
+++ src/bless-0.5.2/configure.ac	2007-09-02 14:30:52.625000000 -0500
@@ -7,8 +7,7 @@
 #
 # I18N
 #
-AM_GNU_GETTEXT([external])
-AM_GNU_GETTEXT_VERSION(0.14)
+AM_GLIB_GNU_GETTEXT
 
 GETTEXT_PACKAGE=bless
 AC_SUBST(GETTEXT_PACKAGE)
